extra/register-service.rb

```diff
@@ -23,10 +23,10 @@
 default_service_name = 'smart proxy'
 
 puts "This service must be run under an account that is a member of 'DHCP Administrators' group"
-puts 'The acount can be local or a domain account. If it is a domain account then use the domain\account syntax'
-service_name = ask("Enter the name of the service ")  {|s| s.default = default_service_name}
-user         = ask("Run this service as this user? ") {|u| u.default = default_user}
-pass1        = ask("Enter the user's password ") {|p| p.echo = "x"}
+puts 'The account can be local or a domain account. If it is a domain account then use the domain\account syntax'
+service_name = ask("Enter the name of the service. ")  {|s| s.default = default_service_name}
+user         = ask("Enter the user to run the service as, or press enter to use the following user: ") {|u| u.default = default_user}
```

Comment:
I suppose not.  I rewrote it like 10 times before I just stayed with that.  Basically, it just needs to be intuitive for us that dont really use windows often.  I suppose

> Enter the user to run the service as:  

would be acceptable.  The original wording made it seem like its a yes or no question that doent accept input, when it does.  Your call.